gym/vector/vector_env.py

```diff
@@ -49,129 +48,112 @@ def __init__(self, num_envs, observation_space, action_space):
-
-        Parameters
-        ----------
-        actions : element of :attr:`action_space`
-            Batch of actions.
-
-        Returns
-        -------
-        observations : element of :attr:`observation_space`
-            A batch of observations from the vectorized environment.
+        """Take an action for each parallel environment.
 
-        rewards : :obj:`np.ndarray`, dtype :obj:`np.float_`
-            A vector of rewards from the vectorized environment.
+        Args:
+            actions: element of :attr:`action_space` Batch of actions.
 
-        dones : :obj:`np.ndarray`, dtype :obj:`np.bool_`
-            A vector whose entries indicate whether the episode has ended.
-
-        infos : list of dict
-            A list of auxiliary diagnostic information dicts from each parallel environment.
+        Returns:
+            Batch of observations, rewards, done and infos
         """
-
         self.step_async(actions)
         return self.step_wait()
 
     def call_async(self, name, *args, **kwargs):
+        """Calls a method name for each parallel environment asynchronously."""
         pass
 
     def call_wait(self, **kwargs):
+        """Calls a method and wait for the results."""
         raise NotImplementedError()
 
-    def call(self, name, *args, **kwargs):
+    def call(self, name: str, *args, **kwargs) -> list[Any]:
         """Call a method, or get a property, from each parallel environment.
 
-        Parameters
-        ----------
-        name : string
-            Name of the method or property to call.
-
-        *args
-            Arguments to apply to the method call.
+        Args:
+            name (str): Name of the method or property to call.
+            *args: Arguments to apply to the method call.
+            **kwargs: Keyword arguments to apply to the method call.
 
-        **kwargs
-            Keywoard arguments to apply to the method call.
-
-        Returns
-        -------
-        results : list
-            List of the results of the individual calls to the method or
-            property for each environment.
+        Returns:
+            List of the results of the individual calls to the method or property for each environment.
         """
         self.call_async(name, *args, **kwargs)
         return self.call_wait()
 
-    def get_attr(self, name):
+    def get_attr(self, name: str):
         """Get a property from each parallel environment.
 
-        Parameters
-        ----------
-        name : string
-            Name of the property to be get from each individual environment.
+        Args:
+            name (str): Name of the property to be get from each individual environment.
+
+        Returns:
+            The property with name
         """
         return self.call(name)
 
-    def set_attr(self, name, values):
+    def set_attr(self, name: str, values: Union[list, tuple, object]):
         """Set a property in each parallel environment.
```

Comment:
parallel environment sounds weird but that's just my opinion

```suggestion
        """Set a property in each sub-environment.
```